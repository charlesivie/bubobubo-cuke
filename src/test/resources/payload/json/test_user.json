{
    "user": {
        "username": null,
        "password": "password",
        "passwordConfirm": "password",
        "enabled": true
    },
    "email": "test@user.com",
    "company": null,
    "firstName": null,
    "lastName": null,
    "id": 1
}