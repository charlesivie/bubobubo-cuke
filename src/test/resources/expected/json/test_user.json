{
    "id" : 1,
    "username": "charlie@gmail.com",
    "email": "test@user.com",
    "company": null,
    "firstName": null,
    "lastName": null
}